core/arch/arm/plat-stm32mp1/drivers/stm32mp1_rcc.c

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Not a strong opinion, but this driver would be fine in the usual drivers directory I think.

Reply:
Few drivers will some be introduced for this platform. Most of them will land in **core/drivers/**, all those that could be platform agnostic. For the few drivers that are really specific to the platform, I located their source in **plat-stm32mp1/drivers/**.

Being in the platform subdir allows to use some short cuts. I.e using a short `RCC_` prefix instead of a larger `STM32MP1_RCC_` in many macros because it should be nicely exported. Also some specific calls will link such platform specific drivers together (shared resources, secure cfg hardening) and I though they were better grouped together this way.

Reply:
ok